### PR TITLE
Update bench bot behavior

### LINKED ISSUE #335: Bot question
Could we somehow have a better strategy with the bot?

I'm not sure how to do that because 1. the "on-demand" can (very) rapidly becomes "I don't care / I forgot that thing" but 2. the constant (huge) reporting also creates noise and makes conversations very of hard to follow...

### PR DESCRIPTION
**Pull request by b-ma**
Fix #335

- split `rust` workflow into `build` and `benchmark`
- `benchmark` workflow now runs only if "/bench" is put into a comment

Hope this works... not sure in which branch the workflow actually runs...

### CODE REVIEW

**Comment by b-ma** [comment]
Convert to draft, this is probably broken, I will recheck that

**Comment by b-ma** [comment]
ok looks good now (hopefuly)

**Comment by orottier** [comment]
Let's see! Thanks
